### PR TITLE
fix(ui5-input): ValueStateMessage width fixed

### PR DESCRIPTION
**Pull request by Martotko**
Fixes: #2257

### CODE REVIEW

File: packages/main/src/Input.js [hunk 1]
```diff
@@ -1104,7 +1104,7 @@ class Input extends UI5Element {
 	get styles() {
 		return {
 			popoverHeader: {
-				"width": `${this._inputWidth}px`,
```
Comment (ilhan007) [inline packages/main/src/Input.js:1107]: having comma at the end of object's key/value is a lint rule, to check for lint errors you can execute yarn lint in the project's root

Comment (Martotko) [reply packages/main/src/Input.js:1107]: Done

**Review by MapTo0 — COMMENTED** [review]
is this working on IE?

**Comment by Martotko** [comment]
> is this working on IE?

Yes, it works

**Review by MapTo0 — APPROVED** [review]

**Pull request merged**